### PR TITLE
Heroupdate

### PR DESCRIPTION
**Pull request by KennethKnows**
need to update responsiveness

### CODE REVIEW

File: src/components/HeroBanner.tsx [hunk 1]
```diff
@@ -1,63 +1,58 @@
...
+    <div
+      className="relative flex flex-col md:flex-row justify-between items-start px-6 md:px-12 w-full h-screen"
       style={{
-        backgroundImage: `url(${heroBanner})`,
-        backgroundSize: 'cover',
-        height: '90vh', // Set the height to 90% of viewport height
+        background: "linear-gradient(-180deg, #7783D2 13%, #F0F0F0 83%)",
       }}
     >
+      <div className="text-left md:px-30 lg:px-48 mt-24 md:mt-32 lg:mt-48">
```
Comment (kynttt) [inline src/components/HeroBanner.tsx]: try to decrease the left padding here to make the whole hero section look balance.

**Comment by KennethKnows** [comment]
Updated the hero page

**Review by kynttt — COMMENTED** [review]
Based on the figma design, it supposed to be much closer than the updated code. And tip, make sure the box-model of  the image and the box model of the contents should be siblings, and nest it on a common parent div, so that you have more control over the siblings, specially on the smaller screens, it doesn't look right. Check the image
sda

**Comment by KennethKnows** [comment]
Update Hero 07/24/2024

**Pull request merged**